packages/flutter_tools/lib/src/ios/ios_workflow.dart

```diff
@@ -71,6 +73,13 @@ class IOSWorkflow extends DoctorValidator implements Workflow {
     }
   }
 
+  /// Whether CocoaPods ran 'pod setup' once where the costly pods' specs are cloned.
+  bool get cocoaPodsInitialized {
```

Comment:
I think it would be more idiomatic Dart to give this a name that starts with a verb, e.g. isCocoaPodsInitialized. That's what the core libraries seem to be doing for boolean getters.